charts/gardener-extension-networking-calico/templates/deployment.yaml

```diff
@@ -1,19 +1,17 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: gardener-extension-networking-calico
+  name: {{ include "name" . }}
   namespace: {{ .Release.Namespace }}
   labels:
-    app.kubernetes.io/name: gardener-extension-networking-calico
-    helm.sh/chart: gardener-extension-networking-calico
-    app.kubernetes.io/instance: {{ .Release.Name }}
+{{ include "commonObjectLabels" . | indent 4 }}
+    high-availability-config.resources.gardener.cloud/type: controller
```

Comment:
Why is this extension of type controller while the cilium extension is modelled as server?

Reply:
This extension does not serve any webhooks while Cilium does.